daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -132,6 +139,13 @@ cmdInspect =
     jsonOpt = switch $ long "json" <> help "Output the raw Protocol Buffer structures as JSON"
     cmd = execInspect <$> inputFileOpt <*> outputFileOpt <*> jsonOpt
 
+cmdInspectMore :: Mod CommandFields Command
```

Comment:
```suggestion
cmdVisual :: Mod CommandFields Command
```

Reply:
Why do we take a DALF and a DAR as an input? I would have expected that you only pass the path to a fat DAR and we will then analyze the main dalf file inside.

Reply:
Yes, dar should be only input to this program. I am thinking of taking the dar and read the manifest and process the dalfs within.